PyPortal/PyPortal_Quarantine_Clock/code.py

```diff
@@ -88,7 +82,7 @@
     if (not refresh_time) or (time.monotonic() - refresh_time) > 3600:
         try:
             print("Getting new time from internet...")
-            pyportal.get_local_time(secrets['timezone'])
+            pyportal.get_local_time(os.getenv("TIMEZONE"))
```

Comment:
The timezone was previously lower case "timezone". 

In the guide - Did it change to "TIMEZONE" in the toml file?

Reply:
Yup, guide is all upper-case entries in settings.toml now.